### ISSUE TITLE
 active keys sayng unblocked

### ISSUE THREAD

**Issue by meatman20yt** [BOLL7708/OpenVR2Key#103]
### Current behavior

i was seting up evrything to play and when i open ''openvr2key'' it just say active key's unblocked i tried what i can find on github but it dinint work(sorry for my speling english is my 2 languace)

### Steps to reproduce the behavior

1.run softweare
2.se error
3. idk what to do

### Expected behavior

i expected to se error quick fix done

### Operating system version

windows 11

### SteamVR version

sorry idk

### Application version

windows 11

### Application location

disc c dowlands

### VR HMD

oculus 3

### VR hardware

oculus 3 controlers

### Software to control

voice mod

### Game

gorilla tag

### Additional information

if it helps 
![image](https://github.com/BOLL7708/OpenVR2Key/assets/161475927/a78c0ad0-530e-47a9-8642-84c0c7209b3f)

**Comment by BOLL7708** [comment]
English is also my second language, welcome to the club.

In any case, which bindings is the application using in SteamVR? If it's not loading the right bindings the application will not detect any input.

Unblocked simply means the SteamVR dashboard has been dismissed, as having the dashboard visible will block input to the application.

**Comment by BOLL7708** [comment]
> i dont have the steem dash board open

I explained why it says unblocked, which is in your original report, I didn't say you had it open.

In any case, you have a modified binding active, depending on what you have done with it, it will work or not. You need to show me what is inside for me to see if anything is weird.

**Comment by meatman20yt** [comment]
where inside

**Comment by meatman20yt** [comment]
you mean the open2vrkey files

**Comment by BOLL7708** [comment]
No, I mean inside the bindings. You need to edit the bindings (which according to the screenshot you have done before) and show what buttons are bound to which actions.

**Comment by meatman20yt** [comment]
i dinint edit anithing my brother tried to fix it buy yeah look  
![image](https://github.com/BOLL7708/OpenVR2Key/assets/161475927/1b37904d-ede6-4f32-adf4-915bedea073e)
![image](https://github.com/BOLL7708/OpenVR2Key/assets/161475927/fdc0b154-883d-462e-a8bd-b6de9078de0c)

**Comment by meatman20yt** [comment]
wrong picture 
![Screenshot 2024-03-02 182244](https://github.com/BOLL7708/OpenVR2Key/assets/161475927/8816e38a-64cc-48c4-80f4-3b60d6aaaa8c)

**Comment by BOLL7708** [comment]
This all looks right, so... what happens when you run it, with the SteamVR dashboard hidden, and then press buttons on the controllers? 

You haven't actually explained what you have tried to do, you just mentioned it says unblocked and I explained why.

**Comment by meatman20yt** [comment]
nothing happens when I press buttons and I am connected to steamvr and the steam vr dashboard is hidden

**Comment by meatman20yt** [comment]
i am presing butons and it just dosent worck i tried binding first but when i press the buton it dosent do anithing

**Comment by BOLL7708** [comment]
Yeah I have no idea. The bindings look OK. Have you tried having a SteamVR game running at the same time?

**Comment by meatman20yt** [comment]
yep

**Comment by meatman20yt** [comment]
look 
![image](https://github.com/BOLL7708/OpenVR2Key/assets/161475927/17dbea80-06c9-4b40-b472-9739aed883bd)

**Comment by SLAY3R990** [comment]
meatman20yt, you probably have the **latest pre-release** version downloaded instead of the **latest secure version**. latest secure version. **go to 0.57 and download that instead.** _also, make sure to delete your other version._ sorry for late reply btw
